proto/raft_cmdpb.proto

```diff
@@ -258,6 +258,8 @@ enum StatusCmdType {
     InvalidStatus = 0;
     RegionLeader = 1;
     RegionDetail = 2;
+    RegionMergeTarget = 3;
+    RegionWakeUp = 4;
```

Comment:
what's used for

Reply:
RegionMergeTarget used for learner peer load merge target.
RegionWakeUp used for waking up the hibernate region.

Reply:
How about just keeping `RegionWakeUp`?